### PR TITLE
Fixed readme bug and added suggestion

### PR DESCRIPTION
**Pull request by TimoCuijpers**
The quick start section in the readme has been fixed and suggestions for more clarity are added with comments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,6 +215,7 @@ To create a new record or update an existing one, you can use the `.save()` meth
 **Cloning a model's instance.**
 ```javascript
 // Return a clone of the model
+const user = new User().show(1);
```
Comment (jobjen02) [inline README.md:218]: const user will be an pending request now. 'await' needs to be added.

File: README.md [hunk 2]
```diff
@@ -45,20 +45,20 @@ class User extends Model {
...
 user.name = 'Jane';
-user.update();
+user.update(); // Or .save()
 
 // Delete the user.
 user.destroy();
 
 // List all users.
-const users = User.index();
+const users = new User().index();
```
Comment (jobjen02) [inline README.md:61]: const users will be an pending request now. 'await' needs to be added.

File: README.md [hunk 3]
```diff
@@ -45,20 +45,20 @@ class User extends Model {
 // Create a new user.
 const user = new User();
 user.name = 'John';
-user.store();
+user.store(); // Or .save()
 
 // Retrieve a user.
-const user = User.show(1);
+const user = new User().show(1);
```
Comment (jobjen02) [inline README.md:51]: const user will be an pending request now. 'await' needs to be added.

**Comment by jobjen02** [comment]
Also, the changelog needs to be updated.

File: README.md (hunk 1 above)
Comment (TimoCuijpers) [reply README.md:218]: Await is only possible when the method where the request is called, is async.

Comment (jobjen02) [reply README.md:218]: The readme is for explanation, not for the full implementation. Right now, the user variable will contain a pending promise, that's why the await needs to be added.

Comment (TimoCuijpers) [reply README.md:218]: Okay!

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@weapnl/js-junction",
-  "version": "0.0.9",
+  "version": "0.0.10",
```
Comment (jobjen02) [inline package.json:3]: Please don't update the version.

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+## v0.0.10
+- Updated Readme.md
```
Comment (jobjen02) [inline CHANGELOG.md:6]: Add this under unreleased.

**Review by jobjen02 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (TimoCuijpers) [reply CHANGELOG.md:6]: Okay

File: package-lock.json [hunk 6]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "@weapnl/js-junction",
-      "version": "0.0.9",
+      "version": "0.0.10",
```
Comment (robinvda) [inline package-lock.json:9]: Don't update this. This gets updated when we make a new version.

Comment (TimoCuijpers) [reply package-lock.json]: I'm sorry, forking is very new for me.

**Review by robinvda — APPROVED** [review]

**Pull request merged**